index.html

```diff
@@ -167,38 +188,68 @@ <h2>About me</h2>
-                    <li><a href="https://www.linkedin.com/in/badru-ndawula-32b371236/"><img src="./images/Linkedin icon.jpg" alt=""></a></li>
-                    <li><a href="https://github.com/PrinceBadru"><img src="./images/github.jpg" alt=""></a></li>
-                    <li><a href="#"><img src="./images/medium.jpg" alt=""><a></li>
-                    <li><a href="#"><img src="./images/angellist.jpg" alt=""></a></li>
-                </ul> 
+            </section>
+            </section>
+
+            <section class="contact"> 
+            
+                <section class="form-section">
+                
+                    <div class="form-description">
+                    <p class="touch">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+                    </div>
+                
+                    <div class="contact-form">
+                 <form action="https://formspree.io/f/mjvzwlqy" method="post">
+                   
+                    <ul>
+                        
+                        <li >
+                            <label for="name">
+                                <input type="text" id="name"  name="user_name" placeholder="Full name" maxlength="30" required>
+                            </label>
+                        </li>
+                        <li>
+                            <label for="mail">
+                                <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+                            </label>
+    
+                        </li>
+                        <li>
+                            <label for="msg">
+                                <textarea name="user_message" id="msg" cols="30" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
+                            </label>
+                        </li>
+                        <li class="button back-image">
+                            <button type="submit" class="btn">Get in touch</button>
+                        </li>
+    
+                    </ul>
+    
+                </form>
+    
+                    </div>
+                </section>
+
+
+            </section>
+
+            <footer> 
+            <ul class ="socials">
+                <li><a href="https://twitter.com/BadruNdawula"><img src="./images/twitter.jpg" alt=""></a></li>
+                <li><a href="https://www.linkedin.com/in/badru-ndawula-32b371236/"><img src="./images/Linkedin icon.jpg" alt=""></a></li>
+                <li><a href="https://github.com/PrinceBadru"><img src="./images/github.jpg" alt=""></a></li>
+                <li><a href="#"><img src="./images/medium.jpg" alt=""><a></li>
+                <li><a href="#"><img src="./images/angellist.jpg" alt=""></a></li>
+            </ul> 
+            
+
+           
+        </footer>
+            
```

Comment:
- [ ] You will notice there is a thin grey line before the social icons. Kindly add it.
- [ ] You will notice there is no mail address under the icon, kindly display it none for desktop version.
## Figma
![image](https://user-images.githubusercontent.com/86200356/193415565-9120e094-20f4-435d-b988-c5fc3855a799.png)